crates/re_space_view/src/data_query.rs

```diff
@@ -45,4 +31,10 @@ pub trait DataQuery {
         ctx: &StoreContext<'_>,
         entities_per_system: &EntitiesPerSystem,
     ) -> DataQueryResult;
+
+    fn execute_query_fast(
```

Comment:
Misses doc string. Also I quite dislike the name: It's not so much fast, it's just skipping the property resolve.
`execute_query_without_property_resolve`?
The other thing that bugs me is that the regular `execute_query` isn't taking `execute_query_fast` as it's input :/